.gitignore

```diff
@@ -1,3 +1,5 @@
+.idea
+*.iml
```

Comment:
You can configure a global gitignore to ignore these files in all your projects, or, alternatively, add them to your local checkout's `.git/info/exclude` file.